frontend/package.json

```diff
@@ -3,6 +3,7 @@
     "version": "0.1.0",
```

Comment:
there shouldn't be such a huge diff on your package-lock.json, let's figure that out first before merging this in. You might have to do a clean install with a node version >=16